20221125-002750
0.11099600791931152
898509
Case 1
Traceback (most recent call last):
  File "prun_train/prune.py", line 254, in <module>
    main(exp,args)
  File "prun_train/prune.py", line 220, in main
    loose_model, prune_mask_dict, prune_bn_dict, mask_bn_channel_dict)
  File "prun_train/prune.py", line 102, in init_weights_from_loose_model
    (pruned_name_temp, pruned_layer_temp) = pruned_model.named_modules()
ValueError: too many values to unpack (expected 2)
done
